### ISSUE TITLE
False positive on Bankig app

### ISSUE THREAD

**Issue by KS1VA** [mullvad/dns-blocklists#141]
### Tried reporting upstream already?

- [X] I have attempted already, but to no avail.

### What are the desired URLs to be unblocked?

alfabank.ru
AlfaBank is just bank in Russia.
There is no wrong with it)

**Comment by oskaralmlov** [comment]
Either it was a false positive in one of the lists that we get via rpz or there was another issue because there is nothing in our history that shows this domain has been blocked.

Either way it is resolvable now:
```
[user@net-mullvad ~]$ dig @all.dns.mullvad.net +https alfabank.ru +short
217.12.104.100
```
